README.md

```diff
@@ -105,4 +105,31 @@ E.g., `url`, `email`, etc.
 
 Doing this communicates semantics to user agents and assistive tech. For example, using `type="numeric"` might trigger the appropriate keyboard state on mobile.
 
+<a name="anchor-tag-usage"></a>
+### Anchor Tag Usage
+
+#### Placeholder Anchor Tags
+```
+<a>Text</a>
+```
+
+Placeholders are great for navigation or displaying hidden information on a page. Using a placeholder makes the element read as clickable to a screenreader so it is the preferred solution over a `<div>` or `<span>`.
+
+This element type first became valid in the HTML5 spec, so make sure your DOCTYPE isn't XHTML before use. If you're stuck in older HTML, use a link anchor tag.
+
+Reference: http://stackoverflow.com/questions/5292343/is-an-anchor-tag-without-the-href-attribute-safe
+
+#### Links
+```
+<a href="www.google.com">Search Google</a>
+```
+
+Use this style of anchor tag to create links to other pages.
+
+If using a link anchor tag as a placeholder (`<a href="#">Not a real link</a>), then be sure to add `ev.preventDefault()` to your JS click event. Otherwise, the page will jump to the top every time a user clicks your placeholder.
```

Comment:
Addressing the two separate issues here:

* The semantic question of link vs button:
  * A button invokes an action: Save, Cancel, Exterminate.
    * Activated with the `Enter` key.
    * Identified in screen readers as a button, and included in the list of form controls.
  * A link takes you somewhere (navigates).
    * Activated with the `Space` key.
    * Identified in screen readers as a link, and included in the list of page links.
* The difference between `<button>` and `<input type="submit/button/etc">`:
  * You can use either. The advantage of `<button>` is that you can put other elements inside it, whereas you can only use plain text labels for `<input>`-style buttons.
  * A `button` is by default a *submit* button, and will trigger the action of a form if it's inside one. Add `type="button"` to make it a non-submit button.
* A method of nullifying an href so a JS event handler can be attached without the link click reloading the page. `href="#"` by itself will just reload the page. `href="javascript://"` (there are other variations on this) will prevent the default link action.

There _are_ ambiguous cases...one common pattern is to present the primary action as a button and the secondary action as a link (e.g., Save button + Cancel link). As with everything, it's not simply a matter of style -- the semantics are important, and have real consequences for keyboard and screen reader users.

BTW, an `<a>` without the href is not new to HTML5, and they are usually just called *anchors*. A lot of times they have an ID so you can link to a document fragment. The term "placeholder link" is new to me, and IMO less accurate than "anchor."